{{cookiecutter.github_project_name}}/src/tsconfig.json

```diff
@@ -1,15 +1,16 @@
 {
   "compilerOptions": {
     "declaration": true,
-    "noImplicitAny": true,
-    "lib": ["dom", "es5", "es2015.promise", "es2015.iterable"],
-    "noEmitOnError": true,
-    "strictNullChecks": true,
+    "lib": ["es2015", "dom"],
     "module": "commonjs",
     "moduleResolution": "node",
-    "target": "ES5",
+    "noEmitOnError": true,
+    "noUnusedLocals": true,
     "outDir": "../lib",
+    "target": "es2015",
+    "strict": true,
     "skipLibCheck": true,
-    "sourceMap": true
```

Comment:
Why is `sourceMap` dropped?

Reply:
@jasongrout was helping me get the cookie-cutter working during the JupyterCon spring day and if I remember correctly, we copied the `tsconfig.json` from the [mimereader-cookiecutter](https://github.com/jupyterlab/mimerender-cookiecutter-ts/blob/master/%7B%7Bcookiecutter.extension_name%7D%7D/tsconfig.json). But I'm not sure if this is necessary to drop `sourceMap`.

Also, I have my custom widget working locally [clustergrammer_widget2](https://github.com/ismms-himc/clustergrammer_widget2/blob/master/examples/introduction.ipynb), but I raised an issue #29 about getting it to work on NBViewer.